docs/how-to-guides/performing-cluster-maintenance.md

```diff
@@ -0,0 +1,239 @@
+# Perform cluster maintenance
+
+MicroCeph provides a simple and consistent workflow to support cluster maintenance activity.
+
+## Prerequisite
+
+Cluster maintenance requires extra redundancy in ceph services, make sure you have at least 4 units of microceph and
+that all units have ceph monitor enabled.
+
+For example, one possible 4 nodes MicroCeph cluster can be like:
+
+```text
+Model      Controller  Cloud/Region         Version  SLA          Timestamp
+microceph  overlord    localhost/localhost  3.6.3    unsupported  14:09:58+08:00
+
+App        Version  Status  Scale  Charm      Channel      Rev  Exposed  Message
+microceph           active      4  microceph  latest/edge  103  no
+
+Unit          Workload  Agent  Machine  Public address  Ports  Message
+microceph/0   active    idle   0        10.42.75.217
+microceph/1*  active    idle   1        10.42.75.46
+microceph/2   active    idle   2        10.42.75.23
+microceph/3   active    idle   3        10.42.75.189
+
+Machine  State    Address       Inst id        Base          AZ  Message
+0        started  10.42.75.217  juju-681fff-0  ubuntu@24.04      Running
+1        started  10.42.75.46   juju-681fff-1  ubuntu@24.04      Running
+2        started  10.42.75.23   juju-681fff-2  ubuntu@24.04      Running
+3        started  10.42.75.189  juju-681fff-3  ubuntu@24.04      Running
+
+MicroCeph deployment summary:
+- juju-681fff-0 (10.42.75.217)
+  Services: mds, mgr, mon, osd
+  Disks: 1
+- juju-681fff-1 (10.42.75.46)
+  Services: mds, mgr, mon, osd
+  Disks: 1
+- juju-681fff-2 (10.42.75.23)
+  Services: mds, mgr, mon, osd
+  Disks: 1
+- juju-681fff-3 (10.42.75.189)
+  Services: mon, osd
+  Disks: 1
+```
+
+## Procedures
```

Comment:
Remove "Procedure". I don't think this title is necessary - it would be okay to jump right to the steps (H2 headers without the numbering).